jira_get_ticket.html

```diff
@@ -9,9 +9,17 @@
 {% block widget_content %} <!-- Main Start Block -->
 
 <!-- File: jira_get_ticket.html
-  Copyright (c) 2016-2021 Splunk Inc.
+  Copyright (c) 2019-2021 Splunk Inc.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
```

Comment:
Are these document changes intentional?

Reply:
why does copyright year changed from 2016 to 2019?

Reply:
These document changes are intentional to solve failed static test: "Copyright". [Fix commit](https://github.com/splunk-soar-connectors/jira/pull/1/commits/0904f748e1f2be65d2b5f847d559b911e82d932a) already included.